internal/common/database/helper.go

```diff
@@ -0,0 +1,65 @@
+package database
+
+import (
+	"strings"
+)
+
+type GetAllPaginationSpec struct {
+	Limit int
+	Page  int
+	Order string
+}
+
+type GetAllPaginationArgs struct {
+	Limit  int
+	Offset int
+	Order  string
+}
+
+type PaginationMetadata struct {
+	TotalPage   int
+	CurrentPage int
```

Comment:
might as well add `TotalEntries` in case we need something like `1 out of 84 shown` type of thing
```suggestion
	CurrentPage int
	TotalEntries int
```